README.md

```diff
@@ -73,7 +73,30 @@ var avrgirl = new Avrgirl({
 });
 ```
 
-If you'd like a quick way of listing out the current available USB ports on your system, run `node lib/list.js` from the `avrgirl-arduino` root within your `node_modules` directory.
+You can list available USB ports programmatically using the the `listPorts` method:
+
+```javascript
+Avrgurl.listPorts(function(err, ports) {
+  console.log( ports );
+  // [ { comName: '/dev/cu.usbmodem1421',
+  // manufacturer: 'Arduino (www.arduino.cc)',
+  // serialNumber: '55432333038351F03170',
+  // pnpId: '',
+  // locationId: '0x14200000',
+  // vendorId: '0x2341',
+  // productId: '0x0043',
+  // _standardPid: '0x0043',
+  // _boardNames: [ 'uno' ] } ]
+});
+```
+
+Alternatively, you can use the CLI:
+
+```
+$ avrgirl-arduino list
+Listing Devices:
+ - port: /dev/cu.usbmodem1421 / pid: 0x0043 / mfg: Arduino (www.arduino.cc) / board: uno
```

Comment:
the forward slashes make this look a little confusing and heard to read IMO. This output also wouldn't pipe super well into anything other than grep. Would perhaps a more minimal JSON object to the raw one suffice instead?